sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreePatterns.scala

```diff
@@ -134,6 +134,7 @@ object TreePattern extends Enumeration  {
   val UNRESOLVED_WINDOW_EXPRESSION: Value = Value
 
   // Unresolved Plan patterns (Alphabetically ordered)
+  val TABLE_VALUED_FUNCTION_WITH_ALIAS: Value = Value
```

Comment:
Since we already have `UNRESOLVED_TABLE_VALUED_FUNCTION`, according to the convention,
`TABLE_VALUED_FUNCTION_WITH_ALIAS` -> `UNRESOLVED_TABLE_VALUED_FUNCTION_WITH_ALIAS`?

Reply:
If we change this, we also need to rename the case class `TableValuedFunctionWithAlias` too.